package-tests.cake

```diff
@@ -279,20 +279,20 @@ AddToBothLists(new PackageTest(
 //    new MockAssemblyExpectedResult("netcore-6.0", "netcore-8.0"),
 //    KnownExtensions.NUnitProjectLoader));
 
-// V2 Result Writer Test
-StandardRunnerTests.Add(new PackageTest(
-    1, "V2ResultWriterTest",
-    "Run mock-assembly under .NET 4.6.2 and produce V2 output",
-    "testdata/net462/mock-assembly.dll --result=TestResult.xml --result=NUnit2TestResult.xml;format=nunit2",
-    new MockAssemblyExpectedResult("net-4.6.2"),
-    KnownExtensions.NUnitV2ResultWriter));
+//// V2 Result Writer Test
+//StandardRunnerTests.Add(new PackageTest(
+//    1, "V2ResultWriterTest",
+//    "Run mock-assembly under .NET 4.6.2 and produce V2 output",
+//    "testdata/net462/mock-assembly.dll --result=TestResult.xml --result=NUnit2TestResult.xml;format=nunit2",
+//    new MockAssemblyExpectedResult("net-4.6.2"),
+//    KnownExtensions.NUnitV2ResultWriter));
 
-StandardRunnerTests.Add(new PackageTest(
-    1, "V2ResultWriterTest",
-    "Run mock-assembly under .NET 6.0 and produce V2 output",
-    "testdata/net6.0/mock-assembly.dll --result=TestResult.xml --result=NUnit2TestResult.xml;format=nunit2",
-    new MockAssemblyExpectedResult("netcore-6.0"),
-    KnownExtensions.NUnitV2ResultWriter));
+//StandardRunnerTests.Add(new PackageTest(
```

Comment:
Commented out code. Does something need fixing?

Reply:
Because of changes in the API, extensions written for version 3 will need to be re-written, which is why almost all of them were already commented-out until I have a chance to look at them individually. For some reason I don't actually understand, the ResultWriter extension continued to work until this latest change. I decided to postpone it until the infrastructure changes are done.